packages/notification-controller/src/NotificationController.ts

```diff
@@ -31,6 +31,13 @@ export type Notification = {
   message: string;
 };
 
+export type NotificationOptions = {
+  message: string;
+  title?: string;
+  detailedView?: string;
```

Comment:
If this is intended to be a snap interface ID, should we name it like so?

```suggestion
  interfaceId?: string;
```

Reply:
yeah, that's fair.